STYLE_GUIDE.md

```diff
@@ -0,0 +1,97 @@
+# panther-analysis Style Guide
+
+## Metadata Best Practices
+
+### RuleID, Filename and DisplayName
+
+- RuleID, Filename and DisplayName should be similar enough to eachother to be able to easily identify a filename by it's RuleID or vice versa.
+- RuleID should start with the LogType idenifier followed by a `.`
+
+```yaml
+DisplayName: "AWS Compromised IAM Key Quarantine"
+RuleID: "AWS.CloudTrail.IAMCompromisedKeyQuarantine"
+Filename: aws_iam_compromised_key_quarantine.py
+```
+
+### Severity
+
+Review the [Alert Severity Guidelines](https://docs.panther.com/detections/rules#alert-severity) in Panther's documentation.  Consider additional factors that could increase or decrease severity, such as exploitation in the wild, potential for false positives, and actionability.
+
+### Reference
+
+Reference should link to a relevant security research or threat report describing the attack being detected and why you'd want to detect it from a security perspective.  Links to API docs or other generic log source documentation should be avoided.
```

Comment:
The Reference value should be a link to a relevant security or threat research report that describes the attack this rule detects, as well as why it is valuable to detect it, from a security perspective. Avoid generic documentation links, such as general API or log source pages.